store/event-store/src/main/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStore.java

```diff
@@ -301,4 +301,45 @@ public void flush() throws Exception {
     public CloseableIterable<String> getTypes(String prefix, Auths auths) {
         return helper.getKeyValueIndex().getTypes(prefix, auths);
     }
+
+    public static class Builder {
```

Comment:
Really?  You're that formal with the merge requests?   I didn't understand how related the EntityStore and the EventStore are.  I'll take another look.  I also fixed the compilation issues, but I'm having issues in my test at query time.